### DISCUSSION TITLE
Playing old expeditions, but it says milestones incomplete and expedition has ended

### DISCUSSION THREAD

**Discussion by Kaostico** [leodium/leodium-ReLive-NoMansSky-Expeditions-Offline#18] [General]
I was playing an old expedition using ReLive method to do so. Well, I closed the game, went online to download a gift my wife sent me on Steam and accidentally started No Man's Sky while online. I spawned on the Anomaly and my expedition was over and ready for termination. I quickly closed the game, went offline, changed the json again in the cache folder and when I started the game, I was able to continue the expedition, but now every milestone shows as INCOMPLETE even if I can keep playing it, it doesn't let me choose what milestone I want to complete, it automatically selects the next milestone in the list.
Is there a way to change the milestones back to normal? Even if it needs some editing on the save file data? It is quite annoying right now.

**Comment by cwmonkey** [comment]
Weird, never heard of this happening. Normally when I do it everything goes back to normal when I go offline and restart the game. I'll check it out now.

**Reply by cwmonkey** [reply]
Actually @Kaostico can you tell me which expedition, and confirm you started the expedition from a main save?

**Reply by Kaostico** [reply]
Of course. It was the Expedition 13: Adrift. I started it from my main save, using the Expedition Console. This bug persisted all the way until I finished the whole expedition. It was kind of annoying because it didn't let me choose what milestones to complete and didn't show what rewards I would get from them until I finish them.
Quite weird I must say but not a hard-lock or anything of the like.

**Reply by cwmonkey** [reply]
I'm testing something else with Leviathan right now, but I checked to see if going online works as expected with a new expedition save and it seems to be fine. I'll try doing the same on a primary save -> adrift expedition save next.

**Reply by cwmonkey** [reply]
Oh another question @Kaostico: Had you restarted the game to try to fix the weird issues that came up?

**Reply by Kaostico** [reply]
Yep, I tried restarting the game the first time. And when I saw I could still play, I just shrugged and kept playing. I finished the rest of the expedition in a couple of sessions, so I had to close the game and start it again. The weird bug never went away.

**Reply by Kaostico** [reply]
Oh, I forgot to mention that I accidentally saved the game the moment the bug appeared because I got into my ship and the out of ir. Perhaps that's what made the bug stay?
